tests/lib/utils/ember-test.js

```diff
@@ -816,34 +822,40 @@ describe('isInjectedServiceProp', () => {
 
     it("should check that it's not an injected service prop when 'service' is not a function", () => {
       const context = new FauxContext(`
+        import {inject as service} from '@ember/service';
         export default Controller.extend({
           currentUser: service.otherFunction()
         });
       `);
-      const node = context.ast.body[0].declaration.arguments[0].properties[0];
-      expect(emberUtils.isInjectedServiceProp(node)).toBeFalsy();
+      const importName = context.ast.body[0].specifiers[0].local.name;
+      const node = context.ast.body[1].declaration.arguments[0].properties[0];
+      expect(emberUtils.isInjectedServiceProp(node, undefined, importName)).toBeFalsy();
     });
   });
 
   describe('native classes', () => {
     it("should check if it's an injected service prop when using renamed import", () => {
       const context = new FauxContext(`
+        import {inject as service} from '@ember/service';
         class MyController extends Controller {
           @service currentUser;
         }
       `);
-      const node = context.ast.body[0].body.body[0];
-      expect(emberUtils.isInjectedServiceProp(node)).toBeTruthy();
+      const importName = context.ast.body[0].specifiers[0].local.name;
+      const node = context.ast.body[1].body.body[0];
+      expect(emberUtils.isInjectedServiceProp(node, undefined, importName)).toBeTruthy();
     });
 
     it("should check if it's an injected service prop when using decorator", () => {
       const context = new FauxContext(`
+        import {inject} from '@ember/service';
         class MyController extends Controller {
           @inject currentUser;
         }
       `);
-      const node = context.ast.body[0].body.body[0];
-      expect(emberUtils.isInjectedServiceProp(node)).toBeTruthy();
+      const importName = context.ast.body[0].specifiers[0].local.name;
+      const node = context.ast.body[1].body.body[0];
+      expect(emberUtils.isInjectedServiceProp(node, undefined, importName)).toBeTruthy();
     });
 
     it("should check that it's not an injected service prop when using another decorator", () => {
```

Comment:
We should add one more test case here that the function returns falsy for `@service foo` without any import.